README.md

```diff
@@ -110,6 +110,7 @@ The following set `P` of types are permitted as parameters to any API method:
     * the set `J`
     * the set `V`    
 * any Enum class
+* the exact `java.lang.Throwable` class
```

Comment:
Worth mentioning that Exception is not supported. Maybe a sentence like this:

`java.lang.Throwable` - must be this class exactly, `Exception` or sub types are not supported.